tests/phpunit/Mutator/AbstractMutatorTestCase.php

```diff
@@ -126,10 +131,12 @@ protected function setUp(): void
 
     final protected function createMutator(array $settings = []): Mutator
     {
-        $class = get_class($this);
-        $mutator = substr(str_replace('\Tests', '', $class), 0, -4);
+        $mutatorClassName = SourceTestClassNameScheme::getSourceClassName(get_class($this));
 
-        return new $mutator(new MutatorConfig($settings));
+        // TODO: this is a bit ridicule...
+        return self::getMutatorFactory()->create([
```

Comment:
As per the TODO: I think this is really bad in terms of DX (for us). It's ok in the source codebase because of the way it is used, but maybe here in the test codebase we can create a helper to make it a bit nicer...